.devcontainer/setup.py

```diff
@@ -83,15 +83,7 @@ def setup():
 
     # FIXME Remove when https://jira.percona.com/browse/PMM-5197 is done.
```

Comment:
let's remove this comment as well

Reply:
I'm not sure but maybe we can remove all setup() functions? We used it only for change encoding